### DISCUSSION TITLE
Dynamic Theme

### DISCUSSION THREAD

**Discussion by billouboq** [tamagui/tamagui#134] [Ideas]
Hi, is it possible to create theme after some HTTP calls ?

We have different Colors for different clients and we need to fetch a configuration before creating the theme, is it possible ?

**Comment by natew** [comment]
@billouboq going to work on this soon.

**Reply by natew** [reply]
Almost ready https://github.com/tamagui/tamagui/pull/160

**Comment by natew** [comment]
Added in Beta 137 https://github.com/tamagui/tamagui/releases/tag/v1.0.1-beta.137

Docs: https://tamagui.dev/docs/intro/themes#dynamic-themes

**Reply by billouboq** [reply]
That’s so cool ! Thanks 🙏 
Gonna test it soon
